lsp-types/generator/CodeGen.hs

```diff
@@ -106,7 +106,13 @@ genModule name pragmas mexports action = do
       -- TODO: replace with regex
       isSelfImport imp = (" " <> fullModName <> " ") `T.isInfixOf` imp || (" " <> fullModName) `T.isSuffixOf` imp
       importSection = hardvcat (fmap pretty $ filter (not . isSelfImport) $ toList imports)
-      mod = warning <> hardline <> pragmaSection <> hardline <> optionsSection <> hardline <> header <> hardline <> hardline <> importSection <> hardline <> hardline <> doc
+      mod = warning <> hardline
+          <> pragmaSection <> hardline
+          <> optionsSection <> hardline
+          <> header <> hardline <> hardline
+          <> importSection 
+          <> hardline <> hardline 
+          <> doc <> hardline
```

Comment:
The actual change is all the way down here: addition of hardline at the end of generated files.

Reply:
Thank you!